core/trino-grammar/src/main/antlr4/io/trino/grammar/sql/SqlBase.g4

```diff
@@ -225,7 +225,7 @@ tableElement
     ;
```

Comment:
This commit is incomplete (and incorrect) without either:
* failing the CREATE when a DEFAULT clause is present
* teaching the engine how to handle the DEFAULT clause

It's ok to keep it separate for review, but you should squash it with the next commit before it gets merged.